introduction.md

```diff
@@ -1 +1,5 @@
-Hello, my name is Judy.  Nice to be here. Favorite path is JavaScript.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?